tests/integration_tests/data_steward/sandbox_test.py

```diff
@@ -0,0 +1,36 @@
+#Python imports
+import unittest
+
+# Project Imports
+from data_steward import sandbox
```

Comment:
This import is failing in the integration test. `data_steward` is the home directory in the python path so we don't need to specify it.
```suggestion
import sandbox
```